modules/rtdModule/index.js

```diff
@@ -200,6 +225,7 @@ function setDataForPrimaryAdServer(data) {
     utils.logError('window.googletag is not defined on the page');
     return;
   }
+  data = validateProviderDataForGPT(data);
```

Comment:
Why not validate the data when it's returned from the adapter instead of here?  When it's done here, it's not possible to issue a warning but which adapter's data is being dropped.

Reply:
because bids don't have the same limit, we don't want to drop data that can be set to bids

we are about to do some changes to this module soon, among them is separate data for bids and GPT. 
When we have this the validation will be right after the data received as you suggest.